pyanaconda/network.py

```diff
@@ -1524,6 +1529,49 @@ def _update_vlan_interfacename_ksdata(devname, ndata):
     if devname != default_ks_vlan_interface_name(ndata.device, ndata.vlanid):
         ndata.interfacename = devname
 
+def update_slaves_onboot_value(devname, value):
+    """Update onboot value in ifcfg files of device slaves
+
+    :param devname: name of device
+    :type devname: str
+    :param value: value of onboot setting
+    :type value: bool
```

Comment:
These can be written as ``:param str devname:`` and ``:param bool value:`` saving two lines! :)